src/styles/partials/components/_search.scss

```diff
@@ -0,0 +1,38 @@
+.dg_search-container {
+  position: relative;
+  width: 100%;
+  height: $base-spacing-unit * 10;
+  color: $black;
+
+  .dg_search-btn {
+    display: block;
+    position: absolute;
+    right: $base-spacing-unit * 2;
+    top: $padding-small;
+    font-size: $base-font-size -2;
+    color: $black;
+    cursor: pointer;
+    background: none;
+    border: none;
+
+    i:hover {
+      color: $blue-lightest;
+      transition: all 0.3s ease 0s;
+    }
+  }
+}
+.dg_search-text-field {
```

Comment:
this will change, if you make the change above to `_dg_search-input`

Reply:
done

Reply:
It looks like you only need padding here.  
![image](https://user-images.githubusercontent.com/1933400/64536209-b1465b80-d2e6-11e9-91d2-f7f91155ef38.png)

Reply:
just so its documented, font is pending convo.